packages/headless/package.json

```diff
@@ -52,7 +52,7 @@
   "dependencies": {
     "@coveo/bueno": "0.45.8",
     "@coveo/relay": "0.7.4",
-    "@coveo/relay-event-types": "7.7.3",
+    "@coveo/relay-event-types": "7.9.7",
```

Comment:
So err: This should have been a breaking change?

Reply:
indeed yeah, these changes should have been a breaking change in the schema.